hooks/useInvitation.ts

```diff
@@ -43,5 +44,32 @@ export default function useInvitation() {
     },
   });
 
-  return { fetchByUser, acceptInvitation, isSuccess: fetchByUser.isSuccess };
+  const createInvitation = useMutation({
+    mutationFn: async ({
+      orgId,
+      receiverEmail,
+      senderId,
+    }: {
+      orgId: number;
+      receiverEmail: string;
+      senderId: string;
+    }) => createInvitationRequest(orgId, receiverEmail, senderId),
+
+    onMutate: async ({ orgId, receiverEmail, senderId }) => {
+      await queryClient.cancelQueries({ queryKey });
+      const previousInvitations = queryClient.getQueryData(queryKey);
+
+      return { previousInvitations };
+    },
+
+    onError: (_error, _variables, context) => {
+      if (context?.previousInvitations) {
+        queryClient.setQueryData(queryKey, context.previousInvitations);
+      }
+    }
```

Comment:
Either these functions should be deleted set the cache of the organisation which has invited this user.